pkg/source/curve/two/constant.go

```diff
@@ -0,0 +1,19 @@
+package two
+
+import (
+	"math/big"
+
+	"github.com/KyberNetwork/kyberswap-dex-lib/pkg/source/curve"
+	constant "github.com/KyberNetwork/kyberswap-dex-lib/pkg/util/bignumber"
+	utils "github.com/KyberNetwork/kyberswap-dex-lib/pkg/util/bignumber"
+)
+
+var (
+	DefaultGas       = curve.Gas{Exchange: 220000}
+	MinGamma         = utils.NewBig10("10000000000")
+	MaxGamma         = new(big.Int).Mul(big.NewInt(5), utils.NewBig10("10000000000000000"))
```

Comment:
I think [MaxGamma should be 2 * 10**16](https://github.com/curvefi/curve-crypto-contract/blob/master/contracts/two/CurveCryptoSwap2.vy#L156C1-L156C10). I realized that when adding contract's safe-check in `newtonY`.

```suggestion
	MaxGamma         = new(big.Int).Mul(big.NewInt(2), utils.NewBig10("10000000000000000"))
```